deploy/osd-curated-operatorsources-revert/fedramp/10-osd-patch-subscription-source.CronJob.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: batch/v1beta1
```

Comment:
Don't need the clusterrole/binding or cronjob. They're legacy.

Reply:
3 files removed